packages/sw-runtime-caching/demo/service-worker.js

```diff
@@ -29,4 +26,5 @@ const route = new goog.routing.RegExpRoute({
 
 const router = new goog.routing.Router();
 router.registerRoute({route});
-router.setDefaultHandler({handler: new goog.runtimeCaching.NetworkFirst()});
+router.setDefaultHandler({handler: new goog.runtimeCaching.NetworkFirst(
+  {networkTimeoutSeconds: 10})});
```

Comment:
Just in terms of demonstrating usage, it doesn't hurt. We may tweak when we revisit/normalize the demos.